server/controllers/streamController.js

```diff
@@ -0,0 +1,63 @@
+const { getAllFlagsData } = require("./flagsController");
+
+let flags = [];
+let clients = [];
+
+(async function() {
+  flags = await getAllFlagsData();
+})();
+
+const handleNewConnection = async (req, res, next) => {
+  const headers = {
+    'Content-Type': 'text/event-stream',
+    'Connection': 'keep-alive',
+    'Cache-Control': 'no-cache'
+  };
+  res.writeHead(200, headers);
+  const data = `data: ${JSON.stringify(flags)}\n\n`;
+
+  res.write(data);
+
+  const clientId = Date.now();
+
+  const newClient = {
+    id: clientId,
+    res
+  };
+
+  clients.push(newClient);
+  req.on('close', () => {
+    console.log(`${clientId} Connection closed`);
+    clients = clients.filter(client => client.id !== clientId);
+  });
+}
+
+const sendUpdate = (req, res, next) => {
+  const updatedFlags = updateFlags(req, [...flags]);
+  if (updateFlags) {
```

Comment:
what is the `if (updateFlags)` doing? Isn't that a function? Did you maybe mean `updatedFlags`?